sal_interfaces/DREAM/DREAM_Commands.xml

```diff
@@ -3,61 +3,98 @@
 <SALCommandSet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://lsst-sal.tuc.noao.edu/schema/SALCommandSet.xsd">
   <SALCommand>
     <Subsystem>DREAM</Subsystem>
-    <EFDB_Topic>DREAM_command_setEnabled</EFDB_Topic>
-    <Description>Enable or disable DREAM.</Description>
-    <item>
-      <EFDB_Name>enabled</EFDB_Name>
-      <Description>DREAM enabled or disabled.</Description>
-      <IDL_Type>boolean</IDL_Type>
-      <Units>unitless</Units>
-      <Count>1</Count>
-    </item>
+    <EFDB_Topic>DREAM_command_resume</EFDB_Topic>
+    <Description>Indicate that DREAM is permitted to resume automated operations.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_openHatch</EFDB_Topic>
+    <Description>Open the hatch if DREAM has evaluated that it is safe to do so.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_closeHatch</EFDB_Topic>
+    <Description>Close the hatch.</Description>
+  </SALCommand>
+  <SALCommand>
+    <Subsystem>DREAM</Subsystem>
+    <EFDB_Topic>DREAM_command_stop</EFDB_Topic>
+    <Description>Immediately stop operations and close the hatch.</Description>
```

Comment:
Will this be different from `SALGeneric_command_abort`?

Reply:
Good question and frankly I don't know. I have asked Tiago about this (he was involved with writing the ICD) and I'll let you know as soon as I have an answer.